lametro/events.py

```diff
@@ -188,6 +188,9 @@ def scrape(self, window=None) :
             # it exists, to the extras dict.
             e.extras = {'guid': event['EventGuid']}
 
+            legistar_api_url = self.BASE_URL + '/events/{0}'.format(event['EventId'])
+            e.add_source(legistar_api_url, note='api')
+
```

Comment:
I moved this up, so that I could easily create `legistar_api_url` and reuse it on line 221.